autoload/neoformat/formatters/typescript.vim

```diff
@@ -18,6 +18,14 @@ function! neoformat#formatters#typescript#prettier() abort
         \ }
 endfunction
 
+function! neoformat#formatters#javascript#prettiereslint() abort
```

Comment:
```suggestion
function! neoformat#formatters#typescript#prettiereslint() abort
```

Reply:
Whoops, sorry overlooked it somehow by copy-pasting, thanks for the catch.